src/Microsoft.Health.Dicom.Benchmark/Retrieve/WadoBenchmark.cs

```diff
@@ -126,7 +111,7 @@ public async Task NewWado()
             scope.ServiceProvider.GetRequiredService<IMetadataStore>(),
             scope.ServiceProvider.GetRequiredService<IETagGenerator>(),
             scope.ServiceProvider.GetRequiredService<IDicomRequestContextAccessor>(),
-            scope.ServiceProvider.GetRequiredService<IDicomTelemetryClient>(),
+            scope.ServiceProvider.GetRequiredService<StoreMeter>(),
```

Comment:
Why are we using a meter for the `Store` feature in the `Retrieve` feature?

Reply:
Agree that it should be a separate meter. I have created a new meter for storing these metrics.